src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.logic.Parser;
+import duke.logic.TaskList;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import javafx.application.Platform;
+
+/**
+ * Duke is a friendly chatbot who will keep track of your tasks
+ * and whether they have been completed.
+ */
+public class Duke {
+
+    private static final String filePath = "data/tasks.txt";
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Duke() {}
+
+    public String init() {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        String response = ui.getWelcome();
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            response = ui.getLoadingError();
+            tasks = new TaskList();
+        }
+        return response;
+    }
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
```

Comment:
Good practice and stuff that people mostly miss is changing this kind of comment